tox.ini

```diff
@@ -13,14 +13,11 @@
 #  and also to help confirm pull requests to this project.
 
 [tox]
+minversion = 3.7.0
 envlist = py{27,35,36,37}
+isolated_build = true
 
 [testenv]
-basepython =
```

Comment:
Why did you remove this?

I think if you leave this in place it will make this an easy-merge.

Reply:
Sure, I can put it back, but it's not necessary. These are the Tox defaults: https://tox.readthedocs.io/en/latest/config.html#conf-basepython
> For example, `py37` means `python3.7`

Reply:
Regardless of the merits of the change, it's unrelated to the point here, which is "Add `pyproject.toml`" so I'd prefer it be removed from the PR. If you feel strongly enough, you can submit a separate PR for it, where we can discuss the merits of the change on its own. But personally, I think it's unnecessary code churn, and I'd be inclined to prefer to leave things as they are.

Reply:
Travis actually fails because of this, and I can repro it locally.

Reply:
I'm also seeing this failure locally, I think it might have to go with the build isolation. Seems like it could be a bug in tox, but I'm inclined to just remove these as unnecessary.